lambdas/src/apis/explore/controllers/explore.ts

```diff
@@ -0,0 +1,181 @@
+  }
+
+  const hotScenesData = await hotSceneCache.get();
+
+  res.status(200).send(hotScenesData);
+}
+
+async function fetchRealmsData(daoCache: DAOCache): Promise<RealmInfo[]> {
+  const statuses = await fetchCatalystStatuses(daoCache);
+
+  return statuses
+    .flatMap((server) =>
+      server.layers.map((layer) => ({
+        serverName: server.name,
+        url: server.url,
+        layer: layer.name,
+        usersCount: layer.usersCount,
+        usersMax: layer.maxUsers,
+        userParcels: layer.usersParcels,
+      }))
+    )
+    .sort((realm1, realm2) => realm2.usersCount - realm1.usersCount);
+}
+
+async function fetchHotScenesData(daoCache: DAOCache, contentClient: SmartContentClient): Promise<HotSceneInfo[]> {
+  const statuses = await fetchCatalystStatuses(daoCache);
+  const tiles = getOccupiedTiles(statuses);
+
+  if (tiles.length > 0) {
+    const scenes = await contentClient.fetchEntitiesByPointers(EntityType.SCENE as any, tiles);
+
+    const hotScenes: HotSceneInfo[] = scenes.map(getHotSceneRecordFor);
+
+    countUsers(hotScenes, statuses);
+
+    const value = hotScenes.sort((scene1, scene2) => scene2.usersTotalCount - scene1.usersTotalCount);
+
+    return value;
+  } else {
+    return [];
+  }
+}
+
+async function fetchCatalystStatuses(daoCache: DAOCache) {
+  const nodes = await daoCache.getServers();
+  const statuses = await fetchStatuses(nodes);
+  return statuses;
+}
+
+function countUsers(hotScenes: HotSceneInfo[], statuses: ServerStatus[]) {
+  statuses.forEach((server) =>
+    server.layers.forEach((layer) => {
+      layer.usersParcels.forEach((parcel) => countUser(parcel, server, layer, hotScenes));
+    })
+  );
+}
+
+function countUser(parcel: ParcelCoord, server: ServerStatus, layer: Layer, hotScenes: HotSceneInfo[]) {
+  const scene = hotScenes.find((it) => it.parcels?.some((sceneParcel) => parcelEqual(parcel, sceneParcel)));
+  if (scene) {
+    scene.usersTotalCount += 1;
+    let realm = scene.realms.find((it) => it.serverName === server.name && it.layer === layer.name);
+    if (!realm) {
+      realm = {
+        serverName: server.name,
+        layer: layer.name,
+        usersCount: 0,
+        usersMax: layer.maxUsers,
+        userParcels: [],
+      };
+      scene.realms.push(realm);
+    }
+    realm.usersCount += 1;
+    realm.userParcels.push(parcel);
+  }
+}
+
+function getOccupiedTiles(statuses: ServerStatus[]) {
+  return [...new Set(statuses.flatMap((it) => it.layers.flatMap((layer) => layer.usersParcels.map((parcel) => `${parcel[0]},${parcel[1]}`))))];
+}
+
+function getHotSceneRecordFor(scene: Entity): HotSceneInfo {
+  return {
+    id: scene.id,
+    name: scene.metadata?.display?.title,
+    baseCoords: getCoords(scene.metadata?.scene.base),
+    usersTotalCount: 0,
+    parcels: scene.metadata?.scene.parcels.map(getCoords),
+    thumbnail: scene.metadata?.display?.navmapThumbnail,
```

Comment:
Someone had suggested that we add a base64 png here, in order to avoid future requests